app/models/wiki.rb

```diff
@@ -104,6 +104,10 @@ def ensure_wiki_exists
     raise InvalidWikiError unless base_url == "https://#{servername}"
   end
 
+  def edits_enabled?
+    ENV["edit_#{domain}"]
```

Comment:
My tests were failing because of this. We need to add the default of en.wikipedia.org to the test environment.